### DISCUSSION TITLE
How to map a PR which updates a Docker image to a Git commit?

### DISCUSSION THREAD

**Discussion by dmorina** [renovatebot/renovate#16844] [Request Help]
I'm trying to add some additional info in the pull requests that Renovate creates, the idea is to be able to answer the question "What's in this docker image?" by attaching the originating Git commit hash responsible for that image build.

I tried to achieve this by adding a label to the Docker image by setting `org.opencontainers.image.source=https://github.com/$org/$repo/tree/$commitHash` and then using a renovate config  ` "prBodyNotes": ["{{#if sourceUrl}}:warning: {{{sourceUrl}}} :warning:{{/if}}"]`. However, this isn't working because it appears like Renovate is looking at the labels for the previous stable version (?).

Any idea on how to display this kind of info in the PRs that Renovate generates?

Thanks!!

**Comment by rarkins** [comment]
Renovate typically looks for a source *repository* for the `sourceUrl` field. There's also a `gitRef` field which could be populated, but it's not currently used in the `docker` datasource in Renovate

**Reply by dmorina** [reply]
Thank you, I opened a PR to populate the `gitRef` https://github.com/renovatebot/renovate/pull/16859. Let me know if the approach makes sense.
